ocp-scripts/import-image-from-dockerhub.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+set -ue
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ODS_CORE_DIR=${SCRIPT_DIR%/*}
+
+echo_done(){
+    echo -e "\033[92mDONE\033[39m: $1"
+}
+
+echo_warn(){
+    echo -e "\033[93mWARN\033[39m: $1"
+}
+
+echo_error(){
+    echo -e "\033[31mERROR\033[39m: $1"
+}
+
+echo_info(){
+    echo -e "\033[94mINFO\033[39m: $1"
+}
+
+ODS_AT_DOCKERHUB=docker.io/opendevstackorg
+IMAGE_NAME=
+IMAGE_VERSION=
+NAMESPACE=ods
+TARGET_IMAGE_STREAM=
+
+function usage {
+    printf "Import dockerhub image/tags.\n\n"
+    printf "This script will ask interactively for parameters by default.\n"
+    printf "However, you can also pass them directly. Usage:\n\n"
+    printf "\t-h|--help\t\tPrint usage\n"
+    printf "\t-v|--verbose\t\tEnable verbose mode\n"
+    printf "\t-on|--namespace\t\tNamespace, e.g. 'ods'\n"
+    printf "\t-in|--imagename\t\tImagename, e.g. 'ods-provisioning-app'\n"
+    printf "\t-it|--targetimagestream\t\tTargetImageName, e.g. 'ods-provisioning-app'\n"
+    printf "\t-iv|--imageversion\t\tImageversion, e.g. '2.x'\n"
+}
+
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+
+    -v|--verbose) set -x;;
+
+    -h|--help) usage; exit 0;;
+
+    -on|--namespace) NAMESPACE="$2"; shift;;
+    -on=*|--namespace=*) NAMESPACE="${1#*=}";;
+
+    -in|--imagename) IMAGE_NAME="$2"; shift;;
+    -in=*|--imagename=*) IMAGE_NAME="${1#*=}";;
+
+    -iv|--imageversion) IMAGE_VERSION="$2"; shift;;
+    -iv=*|--imageversion=*) IMAGE_VERSION="${1#*=}";;
+
+    -it|--targetimagestream) TARGET_IMAGE_STREAM="$2"; shift;;
+    -it=*|--targetimagestream=*) TARGET_IMAGE_STREAM="${1#*=}";;

```

Comment:
Minor nitpick: Elsewhere we use hyphens for multi-word-params, so to align this would be `--target-image-stream`. Also, not a huge fan of multi-letter params with single hyphen (like `-it`) because often it is possible to combine multiple single-hypen-params (e.g. Docker allows you to do `docker exec -it foo bar`, which is equal to `docker exec -i -t foo bar`).

Maybe `s|stream`, `t|tag`, `n|namespace` ,`i|image` would work too?